src/react-components/create-object-dialog.js

```diff
@@ -1,19 +1,37 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
-
 import giphyLogo from "../assets/images/giphy_logo.png";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPaperclip, faTimes } from "@fortawesome/free-solid-svg-icons";
+import styles from "../assets/stylesheets/create-object-dialog.scss";
+import cx from "classnames";
 
 const attributionHostnames = {
-  "giphy.com": giphyLogo
+  "giphy.com": giphyLogo,
+  "media.giphy.com": giphyLogo
 };
 
 const DEFAULT_OBJECT_URL = "https://asset-bundles-prod.reticulum.io/interactables/Ducky/DuckyMesh-438ff8e022.gltf";
+const isMobile = AFRAME.utils.device.isMobile();
```

Comment:
unfortunately this breaks the homepage, which doesn't include aframe. also it's a bad idea to use `AFRAME` without having `import "aframe"` in the file.

Reply:
https://github.com/mozilla/hubs/pull/504

Reply:
actually https://github.com/mozilla/hubs/pull/505